soapboxpb/deployment.proto

```diff
@@ -6,8 +6,32 @@ import "application.proto";
 import "environment.proto";
 
 service Deployments {
+    rpc ListDeployments (ListDeploymentRequest) returns (ListDeploymentResponse) {}
+    rpc GetDeployment (GetDeploymentRequest) returns (GetDeploymentResponse) {}
```

Comment:
This can return a `Deployment` message and the `GetDeploymentResponse` message can then be deleted.